scripts/check-siblings.js

```diff
@@ -68,7 +68,7 @@ function ValidateSentryPackageParameters(packages) {
       if (installedVersion.split('@').length === 2) {
         errorMessages.push("You must specify the version to the package " + installedVersion + ". ( " + installedVersion + "@" + siblingVersion + ")");
       }
-      else if (!installedVersion.endsWith(siblingVersion)) {
+      else if (!installedVersion.endsWith(siblingVersion) && !installedVersion.includes('%3A' + siblingVersion + '#')) {
```

Comment:
I'm not sure if this is correct. I'm using yarn 4.3.1, and it's not setting `npm_config_argv` environment variable, and I don't know the exact syntax of it.